src/passwordcheck.js

```diff
@@ -27,12 +27,26 @@ define([
       PASSWORD_TOO_SHORT: 'PASSWORD_TOO_SHORT'
     };
 
-    function isStrongEnough(password) {
-      // Check for passwords that at least contain a number and an alphabet,
-      // or if alphabets, then at least (minLength + 4) characters long
-      var regexString = '((?=.*[a-zA-Z])(?=.*[0-9])[A-Za-z0-9!@#$%^&*()_+ ]{' + minLength + ',' + (minLength + 4) + '})|([A-Za-z0-9!@#$%^&*()_+ ]{' + (minLength + 4) + ',})';
```

Comment:
> I dont think so, we are still checking the bloomfilter for the same set of words as before. We are just doing the elimination in a different way.

I was looking at the set of words in `human-ans-shorter.txt` as well as the original list in `crackstation-human-alpha-num-spec.txt`. I used [this script](https://gist.github.com/shane-tomlinson/5411b2ce69c53d0473be) to do a quick analysis on the source lists to see how many words in the set were all alpha or all digits.

I was surprised to find none of either, and that the longest word is 12 characters. I think I mis-understood the contents of the shortened list.

IIUC, this is a dictionary of 8-12 character words that contain only a combination of two or more classes of: letters, numbers, and punctuation. Passwords > 12 characters we consider "safe" even if they are all digits, or all numbers. 

If my understanding is correct, we should make two changes:

1. Simplify the logic used in the [shortened dictionary generation script](https://github.com/mozilla/fxa-password-strength-checker/pull/27/files#diff-5530d5d3d3cc3d2cefeda5a217d72961R16). That regexp is complex, perhaps we can simplify it somehow.
1. Update the content server PR to automatically consider passwords of length > 12 to be safe, and [avoid the false-positive case @vladikoff brings up](https://github.com/mozilla/fxa-content-server/pull/2810#issuecomment-131219289).